core/src/main/scala/scalacss/package.scala

```diff
@@ -61,6 +65,13 @@ package object scalacss {
   case class CssKeyframesEntry(name  : KeyframeAnimationName,
                                frames: Map[KeyframeSelector, StyleStream]) extends CssEntry
 
+  case class CssFontFace(fontFamily  : String,
+                         src         : Seq[String],
```

Comment:
Can this every legally be empty? I think a `Seq.empty` would generate invalid CSS right? If so, lets change this to `NonEmptyVector[String]`.